Automatic Validation ended with:
> 2025-03-31T08:22:12.4259541Z ##[error] Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 74 File: mendelson.AS2.installer.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.installer.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 78 File: mendelson.AS2.locale.en-US.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.locale.en-US.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 72 File: mendelson.AS2.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.yaml
 2025-03-31T08:22:12.4278769Z ##[error] Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 74 File: mendelson.AS2.installer.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.installer.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.installer.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 78 File: mendelson.AS2.locale.en-US.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.defaultLocale.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.locale.en-US.yaml Manifest Error: The manifest version in the schema header does not match the ManifestVersion property value in the manifest. Value: 1.9.0 Line: 2, Column: 72 File: mendelson.AS2.yaml Manifest Error: The schema header URL does not match the expected pattern. Value: # yaml-language-server: $schema=https://aka.***/***-manifest.version.1.9.0.schema.json Line: 2, Column: 25 File: mendelson.AS2.yaml

(Automated response - build 1031.)

--- a/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.installer.yaml
+++ b/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.installer.yaml
@@ -1,14 +1,16 @@
 # Created using wingetcreate 1.9.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.9.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.10.0.schema.json
 
 PackageIdentifier: mendelson.AS2
 PackageVersion: 1.1b67
 InstallerType: exe
 InstallerSwitches:
   Silent: -q -nofilefailures
-  SilentWithProgress: -alerts
+  SilentWithProgress: -q -alerts
 Installers:
 - Architecture: x64
+  AppsAndFeaturesEntries:
+  - DisplayName: mendelson opensource as2 b67
   InstallerUrl: https://unlimited.dl.sourceforge.net/project/mec-as2/install_mendelson_opensource_as2_1.1b67.exe?viasf=1
   InstallerSha256: 4D5438F415F8CB1081E1759AD06FCFA7FAD2605E5D9E19512635FAC5DF8CF0DF
 ManifestType: installer

--- a/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.locale.en-US.yaml
+++ b/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.locale.en-US.yaml
@@ -1,5 +1,5 @@
 # Created using wingetcreate 1.9.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.9.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.10.0.schema.json
 
 PackageIdentifier: mendelson.AS2
 PackageVersion: 1.1b67

--- a/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.yaml
+++ b/manifests/m/mendelson/AS2/1.1b67/mendelson.AS2.yaml
@@ -1,5 +1,5 @@
 # Created using wingetcreate 1.9.4.0
-# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.9.0.schema.json
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.10.0.schema.json
 
 PackageIdentifier: mendelson.AS2
 PackageVersion: 1.1b67

